src/structures/Presence.js

```diff
@@ -205,6 +206,18 @@ class Activity {
      * @type {Readonly<ActivityFlags>}
      */
     this.flags = new ActivityFlags(data.flags).freeze();
+
+    /**
+     * Emoji for a custom activity
+     * @type {?Emoji}
+     */
+    this.emoji = data.emoji ? new Emoji(presence.client, data.emoji) : null;
+
+    /**
+     * Creation date of the activity
+     * @type {number}
+     */
+    this.createdTimestamp = data.created_at;
```

Comment:
This should create a new date and use Date#getTime along with the associated createdAt getter instead of direct assignment

Reply:
for other readers: addressed (discussed on discord)